versioncheckr.js

```diff
@@ -125,6 +125,17 @@ module.exports.handler = (event, context, callback) => {
     return Promise.resolve(callback(null, createResponse(202, 'No action to take')));
   }
 
+  const body = pullRequest.pull_request.body;
+  if (body) {
+    const match = /^@version[- ]?checke?r:(\S+)/im.exec(body);
```

Comment:
Don't use `@` since that is a [GitHub mention](https://help.github.com/articles/basic-writing-and-formatting-syntax/#mentioning-people-and-teams) for the [user named "version"](https://github.com/version).

Instead consider: `Version-Checker: skip` as `^Version-Checke?r:\s*(\S+)`

Jim was right, _I cannot help myself_.

Reply:
A signal character for a command is still a good idea. For example, if you wanted to give more than one, or to two different plugins.